### PR TITLE
update updateSelectInput and mapUsersToOptions

### PR DESCRIPTION
**Pull request by Serhiy91**

### CODE REVIEW

File: dist/internal/handlers.js [hunk 1]
```diff
@@ -5,6 +5,6 @@ Object.defineProperty(exports, "__esModule", {
 });
 const updateInput = exports.updateInput = name => ({ updateUI }) => e => updateUI(name, e.target.value);
 
-const updateSelectInput = exports.updateSelectInput = name => ({ updateUI }) => (_, { value }, cb) => updateUI(name, value, cb);
+const updateSelectInput = exports.updateSelectInput = name => ({ updateUI }) => ({ value }, cb) => updateUI(name, value, cb);
```
Comment (Dartv) [inline dist/internal/handlers.js]: Did you not update the source?

Comment (Dartv) [reply dist/internal/handlers.js]: Remember to always run `yarn build` before commit or just simply run `yarn build:watch`

**Comment by Serhiy91** [comment]
@Dartv fixed

**Comment by Dartv** [comment]
I can't actually merge it right now cause it will break my local branch. I think we need to implement version control

**Pull request merged**